### PR TITLE
Items to local file

### PR DESCRIPTION
**Pull request by J3fftw1**
## Testability

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java [hunk 1]
```diff
@@ -112,7 +112,12 @@ public SlimefunLocalization(SlimefunPlugin plugin) {
 		setDefaultMessages("android.scripts.uploaded", "&bUploading...", "&aSuccessfully uploaded your Script!");
 		setDefaultMessage("android.scripts.rating.own", "&4You cannot rate your own Script!");
 		setDefaultMessage("android.scripts.rating.already", "&4You have already left a Rating for this Script!");
-		
+
+		setDefaultMessage("item.rainbow_wool.name", "&5Rainbow Wool");
+		setDefaultMessages("item.rainbow_wool.lore", "", "&dCycles through all Colors of the Rainbow!");
+		setDefaultMessage("item.rainbow_glass.name"), "&5Rainbow Glass");
+		setDefaultMessages("item.rainbow_glass.lore"), "&dCycles through all Colors of the Rainbow!");
```
Comment (WalshyDev) [inline src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]: Syntax issue. You're closing the method after the first string.

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java [hunk 2]
```diff
@@ -112,7 +112,12 @@ public SlimefunLocalization(SlimefunPlugin plugin) {
 		setDefaultMessages("android.scripts.uploaded", "&bUploading...", "&aSuccessfully uploaded your Script!");
 		setDefaultMessage("android.scripts.rating.own", "&4You cannot rate your own Script!");
 		setDefaultMessage("android.scripts.rating.already", "&4You have already left a Rating for this Script!");
-		
+
+		setDefaultMessage("item.rainbow_wool.name", "&5Rainbow Wool");
+		setDefaultMessages("item.rainbow_wool.lore", "", "&dCycles through all Colors of the Rainbow!");
+		setDefaultMessage("item.rainbow_glass.name"), "&5Rainbow Glass");
```
Comment (WalshyDev) [inline src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]: Syntax issue. You're closing the method after the first string.

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java (hunk 1 above)
Comment (WalshyDev) [reply src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]: Also, where's the empty line?

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java (hunk 2 above)
Comment (WalshyDev) [inline src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]:
```suggestion
		setDefaultMessage("item.rainbow_glass.name", "&5Rainbow Glass");
```

**Comment by WalshyDev** [comment]
Also, also, also, *locale file

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java (hunk 1 above)
Comment (J3fftw1) [reply src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]: changed that

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java [hunk 3]
```diff
@@ -112,7 +112,12 @@ public SlimefunLocalization(SlimefunPlugin plugin) {
 		setDefaultMessages("android.scripts.uploaded", "&bUploading...", "&aSuccessfully uploaded your Script!");
 		setDefaultMessage("android.scripts.rating.own", "&4You cannot rate your own Script!");
 		setDefaultMessage("android.scripts.rating.already", "&4You have already left a Rating for this Script!");
-		
+
+		setDefaultMessage("item.rainbow_wool.name", "&5Rainbow Wool");
+		setDefaultMessages("item.rainbow_wool.lore", "", "&dCycles through all Colors of the Rainbow!");
+		setDefaultMessage("item.rainbow_glass.name", "&5Rainbow Glass");
+		setDefaultMessages("item.rainbow_glass.lore"), "", "&dCycles through all Colors of the Rainbow!");
```
Comment (WalshyDev) [inline src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]: Still syntax issue here. You still close the method after the first string.

Always equal the brackets. Right now it's 1 open and 2 close.

Comment (WalshyDev) [reply src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java]: You fixed as I was writing REEEEEEEE

Yeah, now it's ok.

**Pull request closed without merging**